README.md

```diff
@@ -409,6 +418,45 @@ elastic_profile_id: "docker.unit-test"
 It MUST NOT be specified along with `resources`.
 Available in GoCD server since v16.12.0, yaml plugin 0.4.0.
 
+### Artifacts
+
+There are 3 types of artifacts recognized by GoCD. `Build` and `Test` artifacts are stored on the GoCD server.
+The source artifact and the destination on the GoCD server where the artifact must be stored can be specified.
```

Comment:
This sentence is weird. Can you make it `The source and the destination of the artifact stored on the GoCD must be specified.` ?